variables.tf

```
@@ -139,3 +140,9 @@ variable "time_zone" {
   description = "The timezone to use in scheduler"
   default     = "Etc/UTC"
 }
+
+variable "scheduler_job" {
+  type        = any
```

This should be an object based on the expected interface.

```suggestion
  type        = object({
    name = string
  })
```